packages/galeri/src-xpetra/Galeri_HelmholtzFEM2DProblem.hpp

```diff
@@ -206,8 +205,11 @@ Teuchos::RCP<Matrix> HelmholtzFEM2DProblem<Scalar, LocalOrdinal, GlobalOrdinal,
 
     // Insert KE into the global matrix
     for (size_t j = 0; j < numDofPerElem; j++)
-      if (this->Map_->isNodeGlobalElement(elemDofs[j]))
-        this->A_->insertGlobalValues(elemDofs[j], elemDofs, Teuchos::ArrayView<SC>(KE[j], numDofPerElem));
+      if (this->Map_->isNodeGlobalElement(elemDofs[j])) {
+        auto inds   = Kokkos::Compat::getConstArrayView(elemDofs);
+        auto values = Kokkos::Compat::getArrayView(Kokkos::subview(KE, j, Kokkos::ALL()));
```

Comment:
Annoying that there's not a version of this ``getArrayView`` that can do the impl_scalar_type -> SC casting for us in elasticity...